server/svix-server/src/worker.rs

```diff
@@ -959,6 +961,7 @@ pub async fn queue_handler(
                             cache: &cache,
                             op_webhook_sender: &op_webhook_sender,
                             queue_tx: &queue_tx,
+                            webhook_client: webhook_client.clone(),
```

Comment:
I know `clone`ing is cheap for the webhook_client, but I think we can easily have the WorkerContext take a reference to the client instead, no? That way you'd only have to `clone()` it once on 954.

Reply:
Yep. Makes total sense, especially since the `WorkerContext` is already lifetime'd up.
Reworking it to borrow as much as possible.